### PR TITLE
fix: add new logbook app to dockerfile and in some config/documentation files

### PR DESCRIPTION
**Pull request by Millefeuille42**
#6389 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

File: backend/development.ini.sample [hunk 1]
```diff
@@ -82,18 +82,18 @@ basic_setup.session_secret = change_this_value_please!
...
+# - contents/logbook -> allow to create a logbook with time specific entries. It requires content/file to be enabled.
 # - agenda -> add caldav agenda feature (specific process need to be started)
 # - collaborative_document_edition -> add support for collaborative document edition,
 #   add support for collaborative edition server like collabora/LibreOfficeOnline.
 # - share_content -> allow user to share Tracim content with external user
 # - upload_permission -> allow external with permission to upload some file in a space.
 # - gallery -> add gallery app in Tracim
 # default app.enabled config explicit list config:
-; app.enabled = contents/logbook,contents/thread,contents/file,contents/html-document,contents/folder,contents/kanban,contents/todo,agenda,share_content,upload_permission,gallery
+; app.enabled = contents/thread,contents/file,contents/html-document,contents/folder,contents/kanban,contents/todo,contents/logbook,agenda,share_content,upload_permission,gallery
```
Comment (PhilippeAccorsi) [inline backend/development.ini.sample:94]: Just to make sure: in backend code, this is the same list order ?

File: backend/development.ini.sample [hunk 2]
```diff
@@ -532,7 +532,7 @@ website.usage_conditions =
 # Possible events are in the form ENTITY_TYPE.EVENT_TYPE or ENTITY_TYPE.EVENT_TYPE.CONTENT_TYPE where:
 # ENTITY_TYPE  = user_call | user | workspace | content | mention | workspace_member
 # EVENT_TYPE   = created | modified | deleted | undeleted
-# CONTENT_TYPE = html-document | file | thread | folder | comment
+# CONTENT_TYPE = html-document | file | thread | folder | comment | logbook
```
Comment (PhilippeAccorsi) [inline backend/development.ini.sample:535]: I know this is not PR for that but do you know if kanban nee to be in this list ? (or if it is normal to not seen this app)

**Review by PhilippeAccorsi — COMMENTED** [review]
small question :)

File: backend/development.ini.sample (hunk 1 above)
Comment (Millefeuille42) [reply backend/development.ini.sample:94]: Not 100% sure about that, will check and fix

File: backend/development.ini.sample (hunk 2 above)
Comment (Millefeuille42) [reply backend/development.ini.sample:535]: #6399

**Comment by PhilippeAccorsi** [comment]
test in progress

**Comment by PhilippeAccorsi** [comment]
Tested on my laptop with this command:
```
docker run -e DATABASE_TYPE=sqlite -e TRACIM_WEBSITE__BASE_URL=http://192.168.1.157:66666 -p 66666:80 -v /tmp/tracimlog/etc:/etc/tracim -v /tmp/tracimlog/var:/var/tracim algoo/tracim:logbook
```
I can use the Logbook correctly and directly.

**Review by PhilippeAccorsi — APPROVED** [review]

**Comment by PhilippeAccorsi** [comment]
Ready to merge @Millefeuille42 :1st_place_medal:

**Comment by Millefeuille42** [comment]
thanks @PhilippeAccorsi ! 😄

**Pull request merged**